src/ansi-c/library/netdb.c

```diff
@@ -46,6 +56,8 @@ struct hostent *gethostbyaddr(const void *addr, socklen_t len, int type)
 
 /* FUNCTION: gethostent */
 
+#include <netdb.h>
```

Comment:
Missing the Windows variant?

Reply:
I did look this up, but Windows does not appear to have gethostent.